modules/modify-data-object/pkg/utils/parse/clioptions.go

```
@@ -42,6 +44,17 @@ func (c *CLIOptions) GetDebugLevel() zapcore.Level {
 	return zapcore.InfoLevel
 }
 
+func (c *CLIOptions) GetScriptTimeout() time.Duration {
+	if c.Timeout != "" {
+		timeout, err := time.ParseDuration(c.Timeout)
+		if err == nil {
```

Return error instead of silently using the default?